Misc/NEWS.d/next/macOS/2018-10-17-14-36-08.bpo-24658.Naddgx.rst

```diff
@@ -0,0 +1 @@
+Fix read/write on file with a size > 2GB on macOS
```

Comment:
rather than ">", I would prefer to write:

On macOS, fix reading from and writing into a file with a size larger than 2 GiB.